pkg/dump/meta.go

```diff
@@ -0,0 +1,56 @@
+package dump
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/valyala/fasthttp"
+	"time"
+)
+
+var (
+	version string
```

Comment:
move to main package

split to GitCommit and GitBranch

Reply:
Resolved in 87e1bb75e51f6bdb1da9fe675ca1c023e5356ad0